CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+### TLDR;
+
+Before doing anything else, run `cargo build -p lalrpop`!
+
+From now on compile/test with `cargo compile/test -p lalrpop --features=test`.
+
+Just before your final commit, run `sh update_lrgrammar.sh`
```

Comment:
Do we need to run this, or is that something done during package deployment?

Reply:
You need this to build the project with newly generated `lrgrammar.rs` from `lalrpop/src/parser/lrgrammar.lalrpop`. Plain `cargo build` uses `lalrpop/src/parser/lrgrammar.rs` which is unaffected by any changes in `lrgrammar.lalrpop` so so you use this instead of `cargo build`. Once you're done you use `update_lrgrammar.sh` to replace in-tree `lrgrammar.rs` with the newly generated one. I kind of believe this is explained in "Contributor's how to" section